include/openssl/bio.h

```diff
@@ -545,8 +552,10 @@ int BIO_get_shutdown(BIO *a);
 void BIO_vfree(BIO *a);
 int BIO_up_ref(BIO *a);
 int BIO_read(BIO *b, void *data, int len);
+int BIO_read_ex(BIO *b, void *out, size_t outl, size_t *read);
```

Comment:
Ok, sorry for being wishy washy on the subject, but...  would you mind renaming the parameters here as follows?

``` C
int BIO_read_ex(BIO *b, void *data, size_t datal, size_t *read);
```